### DISCUSSION TITLE
[259680]Tales of Maj'Eyal 马基埃亚尔的传说 本体+DLC 密钥+清单

### DISCUSSION THREAD

**Discussion by tiantang111** [pjy612/SteamManifestCache#90] [Share]
```
appid:259680
keys:
259681;726412d152b0c584be0819be6f2a12213d1fb31d0ac9fec795c459ae4c8db807
316910;435a5b510949506e0cf15c88122accf4e286e2efabe6a5d28604b459ec045252
404250;eedb9ee070a5cc93c5196c81f96cf6c1382a93b99dc6de127036c81b537e4a21
590610;1c1f60ece9f8ef50d88ca45d542fce53b41aa73a646817eecacf995482d4bd2f
```
邮箱：2143431350@qq.com
code：E97454D8276EC777

第一次内测码写错了，现在改好了，对不起对不起，麻烦大佬再弄一次
麻烦再问一下，是只要加入内测，就可以获得共享密钥，有了共享密钥就可以直接在steamtools开启的情况下从SteamAppImport里入库steam游戏，然后在steam里边正常下载，不需要手动修改config和清单吗
[Desktop.zip](https://github.com/pjy612/SteamManifestCache/files/13712852/Desktop.zip)

**Comment by pjy612** [comment]
查收

**Reply by tiantang111** [reply]
我傻了，对不起，我怕写错，抄别人格式的时候好像把内测码也抄上去了，我的是这个：E97454D8276EC777
对不起，对不起，检查了半天，结果还是错的
麻烦再问一下，是只要加入内测，就可以获得共享密钥，有了共享密钥就可以直接在steamtools开启的情况下从SteamAppImport里入库steam游戏，然后在steam里边正常下载，不需要手动修改config和清单吗，我是纯小白，确实不太明白这些东西

**Reply by pjy612** [reply]
看共享密钥的储备量。如果提示没有的话 还是得自备密钥。
不过 密钥格式可以对照 heyong 库的 格式 放到 SAI旁边就行。
不用自己去config.vdf

**Reply by tiantang111** [reply]
所以共享密钥该如何获取呢，还是说大佬你发给我的那个就是，那具体该如何使用呢
我第一次的内测码应该是对的，不知道为什么，运行程序后内测码又变了，现在变成了第一次的D7249C041AEDC982，这，这是正常情况吗?
大佬有对应的群吗，不懂的东西有点多，感觉得去群里学习一下

**Reply by pjy612** [reply]
你是不是 WiFi 设置了 什么随机硬件地址？
lic 文件 解压后放 SAI 旁边就行。
